### ISSUE TITLE
Create icon shortcuts from inside the app

### ISSUE THREAD

**Issue by Acce0ss** [Acce0ss/shellex#3]
Add functionality to create an entry with icon to the app launcher from the existing commands inside the app as described here: https://together.jolla.com/question/24190/launch-shell-commands-from-user-interface/?answer=31474#post-id-31474

Possibly add stop functionality by clicking the icon again.

**Comment by gaetano-guerriero** [comment]
I'm thinking about working on this. Are you willing to accept a patch ?

**Comment by Acce0ss** [comment]
Certainly! I would highly appreciate any work on it. Just put forth any ideas, I'll do a pull if everything seems to work alright
